apps/cli/src/commands/environment/list.environment.ts

```diff
@@ -24,9 +25,39 @@ export class ListEnvironment extends BaseCommand {
     ]
   }
 
-  async action({ args }: CommandActionData): Promise<void> {
+  getOptions(): CommandOption[] {
+    return [
+      {
```

Comment:
Just a suggestion - remove this entire block into the `util` folder into a file named `pagination-options.ts`. The reason being, this will be reused by multiple commands